lib/cql/client/prepared_statement.rb

```diff
@@ -150,7 +150,13 @@ def self.prepare(cql, execute_options_decoder, connection_manager, logger)
       def execute(*args)
         connection = @connection_manager.random_connection
         if connection[self]
-          run(args, connection)
+          f = run(args, connection)
+          f.fallback do |e|
+            raise e unless e.is_a?(Cql::QueryError) && e.code == 0x2500 # unprepared
```

Comment:
It seemed like a convention to use comments, see https://github.com/iconara/cql-rb/blob/master/lib/cql/protocol/responses/detailed_error_response.rb#L16